### ISSUE TITLE
Dark mode: Issue filing options' text fails contrast when the app is opened in light mode and then color setting is changed to dark mode

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#828]
**Describe the bug**
The text of the radio buttons does not switch to  appropriate color when the color setting is changed to dark mode after the app is opened in light mode

**To Reproduce**
1. Make sure color setting for default app mode is set to 'Light'
![image](https://user-images.githubusercontent.com/22068434/86277575-c77e1a80-bb8b-11ea-8029-d539ea7a081e.png)

2. Launch AI-Windows
3. Go to settings -> Connection
4. Go back to windows color settings and now change the default app mode to 'dark'
5. Observe the color of text adjacent to the radio buttons remains dark even though the background is changed to dark

**Expected behavior**
The text color should change to an appropriate color with sufficient contrast against the changed background

**Actual behavior**
The text color remains same as in regular(light) mode

**Screenshots or .GIF**
![dark-1](https://user-images.githubusercontent.com/22068434/86278484-52134980-bb8d-11ea-9ee7-ee65ec9b2c23.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by adarshrema** [comment]
Needs investigation by the PM and design teams

**Comment by DaveTryon** [comment]
@iamrafan, this is fixed in the release at https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1382.01. Please regress/close at your convenience.

**Comment by iamrafan** [comment]
Validated the fix in Canary Release: v1.1.1383.01
